spacy/util.py

```
@@ -555,6 +564,29 @@ def read_jsonl(file_path):
+
+    file_path (unicode / Path): The path to the output file.
+    lines (list): The JSON-serializable contents of each line.
+    """
+    data = [json_dumps(line) for line in lines]
+    with Path(file_path).open("w", encoding="utf-8") as f:
+        f.write("\n".join(data))
+
+
+def is_json_serializable(obj):
+    """Check if a Python object is JSON-serializable."""
+    if hasattr(obj, "__call__"):
```

Ruff (0.12.2)

Using `hasattr(x, "__call__")` to test if x is callable is unreliable. Use `callable(x)` for consistent results.

Replace with `callable()`

(B004)